cluster-scope/overlays/nerc-ocp-obs/secretstores/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- acct-mgt
```

Comment:
You don't need this (we're not running the account management tools on the obs cluster).

Reply:
i wasn't sure, if needed for different right settings than the infra (outside vpn).
following your input here.